viewer/packages/sector-parser/src/GltfSectorParser.ts

```diff
@@ -0,0 +1,165 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+import { MeshGPUInstancing, InstancedMesh } from '@gltf-transform/extensions';
+import { Node, NodeIO, Document, Primitive } from '@gltf-transform/extensions/node_modules/@gltf-transform/core';
```

Comment:
```suggestion
import { Node, NodeIO, Document, Primitive } from '@gltf-transform/core';
```

Reply:
Sadly enough this will brake it. Not sure if it is the lib that is braking or what...